### PR TITLE
updated the depreciation warnings for functions now found in from_lims

### PR DESCRIPTION
**Pull request by DowntonCrabby**
updated the depreciation warning to include October 29th as the date of removal for many depreciated functions.

### CODE REVIEW

File: visual_behavior/data_access/loading.py [hunk 1]
```diff
@@ -1143,7 +1143,7 @@ def get_stim_metrics_summary(behavior_session_id, load_location='from_file'):
 
 # FROM LIMS DATABASE
 # this portion is depreciated, please use functions in from_lims.py instead
-gen_depr_str = 'this function is deprecated and will be removed in a future version, ' \
+gen_depr_str = 'this function is deprecated and will be removed on October 29th 2021, ' \
```
Comment (yavorska-iryna) [inline visual_behavior/data_access/loading.py:1146]: why is this function deprecated? just trying to catch up. thanks.

File: visual_behavior/data_access/loading.py [hunk 2]
```diff
@@ -1428,6 +1428,9 @@ def clean_roi_locations_column_labels(roi_locations_dataframe):
     Returns:
         dataframe -- [description]
     """
+    warn_str = gen_depr_str.format('from_lims',
```
Comment (yavorska-iryna) [inline visual_behavior/data_access/loading.py:1431]: what's the "warn_str" in this case is?

**Review by yavorska-iryna — APPROVED** [review]

File: visual_behavior/data_access/loading.py (hunk 1 above)
Comment (DowntonCrabby) [reply visual_behavior/data_access/loading.py:1146]: All the functions with this depreciation warning have been moved to the from_lims module of loading. So they're still accessible there, but are being removed from the more general loading.py file.

File: visual_behavior/data_access/loading.py (hunk 2 above)
Comment (DowntonCrabby) [reply visual_behavior/data_access/loading.py:1431]: each function has it's own depreciation warning string (warn_str) that's basically a customization of the general depreciation warning string (gen_depr_str). So it gives the general warning "This function is depreciated and will be removed on October 29th 2021. Please use {}.{} instead'   

So it tells you when the function will be removed from the loading.py file and also where you can find the replacement function.

**Pull request merged**